### PR TITLE
Issue#27: Add references to projection note

### PR DESCRIPTION
**Pull request by EN-Chou**
This commit works on issue #27, add references to projection note.

### CODE REVIEW

File: projection.tex [hunk 1]
```diff
@@ -113,7 +113,7 @@ \subsection{Vector form}
 \nabla \cdot \mathbf{v}=0 \label{e:continuity_vec}
 \end{align}
 %
-\section{Projection algorithm}
```
Comment (yungyuc) [inline projection.tex:116]: References should be added in paragraphs instead of section headers.

File: projection.tex [hunk 2]
```diff
@@ -113,7 +113,7 @@ \subsection{Vector form}
...
+\section{Projection algorithm \cite{ferziger_cfd_2019}}
 Velocity at $n^{th}$ time-step is denoted as $\mathbf{v}^n$. Convection term is
```
Comment (yungyuc) [inline projection.tex:117]: Please write something about the numerical method you are employing in this note.  Conventionally we (scientists and engineers) call them numerical methods, not algorithms.  The term algorithm is mostly used by programmers.

**Review by yungyuc — CHANGES_REQUESTED** [review]

File: projection.tex [hunk 3]
```diff
@@ -122,73 +126,83 @@ \section{Projection algorithm}
 -\nabla p+\mathbf{D}(\mathbf{v}^{{n}}) \label{e:momentum_pj} 
 \end{align}
 %
-Solve for $\mathbf{v}^{n+1}$, we need to decide $\mathbf{P}$ beforehand. Projection 
-algorithm is implemented with intermediate term $\mathbf{v}^{*}$, the momentum 
-equation is split into 2 parts: 
+Projection method\cite{ferziger_cfd_2019} split eq (\ref{e:momentum_pj}) into
```
Comment (yungyuc) [inline projection.tex:129]: Conventionally we add citation at the end of a sentence.  But at the time being it is OK to leave it here.  We can come back here to change later.

Comment (yungyuc) [reply projection.tex:129]: And I wonder if you can pin-point to the original paper for the Projection Method?

**Review by yungyuc — APPROVED** [review]

**Comment by yungyuc** [comment]
@EN-Chou I updated your permission and you should be able to merge the PR yourself.  Could you please give it a try?

File: projection.tex (hunk 3 above)
Comment (EN-Chou) [reply projection.tex:129]: Sure, we can change it later; yes, this sounds more reasonable to me. Should they be included in a new issue?

Comment (yungyuc) [reply projection.tex:129]: Sounds like a plan.  Could you file an issue for yourself?

**Pull request merged**